src/components/trial/TrialScreen.tsx

```diff
@@ -39,6 +42,10 @@ const TrialScreen = (props: TrialScreenProps) => {
     }
   };
 
+  const progressClick = () => {
+    if (trialStage == 2) progressTrial();
```

Comment:
Consider using type safe comparison ( === )
Also consider putting the code inside brackets { }